lib/src/bloc/dice_bloc.dart

```diff
@@ -0,0 +1,62 @@
+import 'dart:async';
+import 'dart:math';
+import 'package:flutter/material.dart';
+import 'i_dice_bloc.dart';
+import 'package:dice_roller/utils/assets.dart';
+
+class DiceBloc extends IDiceBloc {
+  bool _dicesAreEqual = true;
+
+  StreamController<List<AssetImage>> _valuesStreamController =
+      StreamController<List<AssetImage>>.broadcast();
+
+  StreamController<bool> _areEqualStreamController =
+      StreamController<bool>.broadcast();
```

Comment:
Remove this unused StreamController and all relative implementation of it from the file.